pallets/bootstrap/src/lib.rs

```diff
@@ -826,12 +870,22 @@ impl<T: Config> Pallet<T> {
 				.checked_add(first_token_rewards)
 				.ok_or(Error::<T>::MathOverflow)?;
 			if non_vested_rewards > 0 {
-				<T as Config>::RewardsApi::activate_liquidity_tokens(
+				let activate_result = <T as Config>::RewardsApi::activate_liquidity_tokens(
 					&who,
 					liq_token_id.into(),
 					non_vested_rewards,
-				)
-				.map_err(|_| Error::<T>::TokensActivationFailed)?;
+				);
+				if activate_result.is_err(){
+					log!(
+						error,
+						"Activating liquidity tokens failed upon bootstrap claim rewards = ({:?}, {}, {})",
+						who,
+						liq_token_id,
+						non_vested_rewards
+					);
+					
+					Self::deposit_event(Event::RewardsLiquidityAcitvationFailed(who.clone(), liq_token_id, non_vested_rewards));
+				};
```

Comment:
consider logging activate result error

```suggestion



				if let Err(err) = let activate_result = <T as Config>::RewardsApi::activate_liquidity_tokens(
					&who,
					liq_token_id.into(),
					non_vested_rewards,
				){
					log!(
						error,
						"Activating liquidity tokens failed upon bootstrap claim rewards = ({:?}, {}, {}), err = {:?}",
						who,
						liq_token_id,
						non_vested_rewards,
						err
					);
					
					Self::deposit_event(Event::RewardsLiquidityAcitvationFailed(who.clone(), liq_token_id, non_vested_rewards));
				};
```

Reply:
Good point. will do